plugins/Makefile.am

```diff
@@ -4,31 +4,28 @@
 # Put Together AM_CPPFLAGS and AM_LDFLAGS
 include $(top_srcdir)/lib_flags.am
 
-AM_LDFLAGS += -module -avoid-version -shared -export-dynamic	\
- -rpath ${abs_builddir} ${NOUNDEFINED}
-
-# Create an alternate directory if not installing or for noinst installs.
-ALTPLUGINDIR = ${abs_top_builddir}/plugins/plugindir
-
-# This is where the plugins are to be installed
-if ENABLE_PLUGIN_DIR
+#if ENABLE_PLUGIN_DIR
 plugindir = @PLUGIN_INSTALL_DIR@
-else
-plugindir = ${ALTPLUGINDIR}
-endif
+#else
+plugindir = ${prefix}
```

Comment:
There's nothing tying `--enable-plugins` to `--with-plugin-dir`; I think just specifying `--enable-plugins` without `--with-plugin-dir` will put the plugins into `${prefix}`.  Would `${libexecdir}/netcdf-c` or similar make more sense for the fallback location, would a better fallback be not installing those plugins at all (as implied by the old value), or is there some reasoning for this choice I'm not aware of?

Reply:
New version is basically what I ended up with.